analyses/cell-type-ewings/exploratory_analysis/06-tumor-cell-state-assignment.Rmd

```diff
@@ -0,0 +1,606 @@
+      ggplot(aes(x = UMAP1, y = UMAP2, color = cluster_classification)) +
+      geom_point(alpha = 0.5, size = 0.1) +
+      theme(
+        aspect.ratio = 1
+      ) +
+      labs(title = id) +
+      scale_color_brewer(palette = "Dark2") +
+      guides(color = guide_legend(override.aes = list(alpha = 1, size = 1.5)))
+  })
+```
+
+Looking at this I think the cells that are classified as tumor cells using the cluster assignments mostly line up with the `SingleR` annotations. 
+So we can label any cells from clusters that have high tumor marker gene expression as tumor cells. 
+
+### Look at marker gene expression for tumor cell states in all cells
+
+Below we look at the expression of genes for the tumor cell states across all cells. 
+This will show us how the expression of these gene sets varies across cells in each library and help ensure we have the correct population of tumor cells.
+
+The value for each of the marker gene sets corresponds to the sum of the expression of all genes in the given gene list. 
+(Note that I also looked at mean and it had similar output)
+
+The gene sets that we use are: 
+
+- `EWS-low`: Marker genes from `tumor-cell-state-markers.tsv` that correspond to the `EWS-low` cell state. 
+- `EWS-high`: Marker genes from `tumor-cell-state-markers.tsv` that correspond to the `EWS-high` cell state. 
+- `proliferative` : Marker genes from `tumor-cell-state-markers.tsv` that correspond to the `proliferative` cell state. 
+Note that a lot of the Ewing sarcoma literature argues that `EWS-FLI1` high cells are proliferative. 
+- `aynaud-ews-high`: Marker genes from `gene_signatures/aynaud-ews-targets.tsv` that correspond to genes activated by `EWS-FLI1`. 
+- `wrenn-ews-low`: Marker genes from `gene_signatures/wrenn-nt5e-genes.tsv` that correspond to genes repressed by `EWS-FLI1`. 
+
+```{r, fig.height=7, fig.width=10}
+# pull out columns that have sum of marker gene expression 
+marker_gene_cols <- colnames(cluster_classification_df)[stringr::str_detect(colnames(cluster_classification_df), "_sum$")]
+
+# look at the expression of all gene sets across both libraries 
+marker_gene_cols |>
+  purrr::map(\(geneset){
+    cluster_classification_df |>
+      ggplot(aes(x = UMAP1, y = UMAP2, color = !!sym(geneset))) +
+      geom_point(alpha = 0.5, size = 0.1) +
+      facet_wrap(vars(library_id)) +
+      scale_color_viridis_c() +
+      theme(
+        aspect.ratio = 1
+      )
+  }) |> 
+  patchwork::wrap_plots(ncol = 2)
+```
+
+It looks like there's a cluster of cells in both libraries that have expression of the `EWS-low` and `wrenn-ews-low` signatures. 
+Those cells correspond to the cells that we are labeling as "chondrocyte" in `SCPCL000822` and "mesenchymal-like" in `SCPCL000824`. 
+The chondrocytes also express high markers of mesenchymal-like genes and the literature talks about tumor cells with low `EWS-FLI1` expression being "MSC-like". 
+I think there's a possibility that these are tumor cells, so let's relabel them. 
```

Comment:
This seems like a pretty decent result here!

Reply:
I agree! And I also was revisiting [this issue](https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/668) that I filed about using these samples to come up with a base reference for SingleR. One of the reasons I was worried about using just SingleR the way we set it up was because a lot of cells were getting called tumor cells that had low expression of the EWS markers. But if they have low expression of the EWS targets and high expression of the EWS low signature then I think those are actually tumor cells! 

One idea I have as follow up here is to label these two samples as I'm trying to do here and revisit using them with SingleR. Probably need a third sample but that might be a good route.